src/accumulator.jl

```diff
@@ -12,6 +12,14 @@ counter(T::Type) = Accumulator(T,Int)
 Accumulator{T,V<:Number}(dct::Dict{T,V}) = Accumulator{T,V}(copy(dct))
 counter{T}(dct::Dict{T,Int}) = Accumulator{T,Int}(copy(dct))
 
+"""
+    counter{T}(seq::AbstractArray{T})
+
+### Args:
```

Comment:
As I noted on the StatsBase PR, I think in most cases the argument list isn't useful. Here, `seq` is mentioned below anyway, and the signature gives the expected type.